TLM/TMPE.API/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 // associated with an assembly.
 [assembly: AssemblyTitle("TMPE.API")]
 [assembly: AssemblyProduct("TMPE.API")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("API assembly to help integration of other Cities: Skylines modifications with TMPE")]
```

Comment:
I have a feeling that all this is for nuget package creation.

Reply:
wb!

Reply:
It can be filled manually. Some day™️  we will release it as nuget package